### PR TITLE
CSVS-109 Integration tests were implemented for read/write policies

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala [hunk 1]
```diff
@@ -20,4 +20,5 @@ package com.bwsw.cloudstack.vault.server.vault
 
 object Constants {
   val tokenNotFoundStatus = 403
+  val policyNotFoundStatus = 404
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala]: remove "status" from the name

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**